packages/app-store/utils.ts

```diff
@@ -108,14 +109,17 @@ export function getLocationGroupedOptions(integrations: ReturnType<typeof getApp
  * This should get all available apps to the user based on his saved
  * credentials, this should also get globally available apps.
  */
-function getApps(userCredentials: CredentialData[]) {
-  const apps = ALL_APPS.map((appMeta) => {
-    const credentials = userCredentials.filter((credential) => credential.type === appMeta.type);
+function getApps(credentials: CredentialData[], filterOnCredentials?: boolean) {
+  const apps = ALL_APPS.reduce((reducedArray, appMeta) => {
+    const appCredentials = credentials.filter((credential) => credential.type === appMeta.type);
+
+    if (filterOnCredentials && !appCredentials.length && !appMeta.isGlobal) return reducedArray;
```

Comment:
`getEnabledApps` relies on `getApps`. By adding the filter credentials in this function, we can include globally installed apps